web/packages/test/scripts/configure-bridgehub.sh

```diff
@@ -15,26 +14,24 @@ config_beacon_checkpoint()
 wait_beacon_chain_ready()
 {
     local initial_beacon_block=""
-    while [ -z "$initial_beacon_block" ] || [ "$initial_beacon_block" == "0x0000000000000000000000000000000000000000000000000000000000000000" ]
-    do
+    while [ -z "$initial_beacon_block" ] || [ "$initial_beacon_block" == "0x0000000000000000000000000000000000000000000000000000000000000000" ]; do
         echo "Waiting for beacon chain to finalize to get initial block..."
-        initial_beacon_block=$(curl -s "$beacon_endpoint_http/eth/v1/beacon/states/head/finality_checkpoints" \
-            | jq -r '.data.finalized.root' || true)
+        initial_beacon_block=$(curl -s "$beacon_endpoint_http/eth/v1/beacon/states/head/finality_checkpoints" |
+            jq -r '.data.finalized.root' || true)
         sleep 3
     done
 }
 
-fund_accounts()
-{
+fund_accounts() {
     echo "Funding substrate accounts"
     transfer_balance $relaychain_ws_url "//Charlie" 1013 1000000000000000 $assethub_sovereign_account
+    transfer_balance $relaychain_ws_url "//Charlie" 1013 1000000000000000 $TEMPLATE_AGENT_ID
```

Comment:
Is this correct?
```suggestion
    transfer_balance $relaychain_ws_url "//Charlie" 1013 1000000000000000 $template_sovereign_account
```

Reply:
Yes, we derivate the agent account same way as `agent_id`, for the reason and more detail in https://github.com/Snowfork/snowbridge/pull/940#discussion_r1308520658